calculatorUICOOP/ViewModels/MainPageViewModel.cs

```diff
@@ -106,28 +118,53 @@ public void DeleteLastInput()
             {
                 ClearScreen();
             }
-           
+
         }
 
-        public void ShowRemainderOnDisplay(string remainder)
+        public void ShowPercentOnDisplay(string percent)
         {
-
+            AssignOperator(percent);
         }
 
         public void ShowDecimalOnDisplay(string decimalDot)
         {
-
+            if (!_hasDecimal)
+            {
+                DisplayContent += decimalDot;
+                _hasDecimal = true;
+            }
         }
 
         public void Equals()
         {
-
+            double _number2 = Convert.ToDouble(DisplayContent);
+            switch (_operator)
+            {
+                case "+":
+                    DisplayContent = MathLogic.Add(_number1, _number2).ToString();
+                    break;
+                case "-":
+                    DisplayContent = MathLogic.Subtract(_number1, _number2).ToString();
+                    break;
+                case "*":
+                    DisplayContent = MathLogic.Multiply(_number1, _number2).ToString();
+                    break;
+                case "/":
+                    if (DisplayContent != "0")
+                        DisplayContent = MathLogic.Divide(_number1, _number2).ToString();
+                    else
+                        DisplayContent = "Can't Divide by 0";
+                    break;
+                case "%":
```

Comment:
Percent should not happen at on Equals press. But rather when % is pressed

Reply:
@srjheam Would you like to do this? This is mainly what it keeping the PR from approval.

Reply:
> @srjheam Would you like to do this? This is mainly what it keeping the PR from approval.

Well, the percent operator is not working properly for two reasons:

1. It can be interpreted by the user in many ways:
1.1. Return the rest of the division by two numbers;
1.2. Return the percentage of a number (Most strange interpretation in my opinion);
2. When its expected behave is to return the percentage of the entered number (which is the case), it doesn't return the percentage, on the contrary it asks for and second input.

I'll fix the second item.